share/logcfg.dat

```diff
@@ -129,6 +129,8 @@ CWTONE=800
 #           for ten tec OMNI VI #
 #################################
 #
+#FOLLOW_MODE
```

Comment:
I suggest a different keyword here, eg. `RIG_FOLLOW_MODE` - what do you think?

Reply:
Let's wait till the end. ;-)